package.json

```diff
@@ -6,24 +6,28 @@
     "@emotion/styled": "^11.6.0",
     "@material-ui/core": "^4.12.3",
     "@material-ui/icons": "^4.11.2",
+    "@mui/icons-material": "5.3.1",
     "@mui/material": "^5.3.0",
     "@reduxjs/toolkit": "^1.7.1",
     "@testing-library/jest-dom": "5.16.1",
     "@testing-library/react": "12.1.2",
     "@testing-library/user-event": "13.5.0",
     "react": "^17.0.2",
+    "react-bootstrap": "2.1.1",
     "react-countup": "^6.1.1",
     "react-dom": "^17.0.2",
+    "react-facebook-login": "4.1.1",
     "react-hook-form": "^7.25.1",
     "react-redux": "^7.2.6",
     "react-router-dom": "^6.2.1",
     "react-scripts": "5.0.0",
     "react-select": "^5.2.2",
+    "react-social-login-buttons": "3.6.0",
```

Comment:
Wydaje mi się, że nie jest to używane w aplikacji

Reply:
usunięte